src/main/java/UseCases/IngredientStorageResponseImpl.java

```diff
@@ -0,0 +1,29 @@
+package UseCases;
+
+import Entities.Ingredient;
+import Entities.Recipe;
+
+import java.util.Collection;
+import java.util.List;
+
+public class IngredientStorageResponseImpl extends AbstractResponse {
+
```

Comment:
I don't think this is the way to go. Our use cases will be tied to returning only `Response` by their interface. So a consumer cannot safely cast down to a sub-class, instead all info should be generically available throughs some means, maybe like the command. Depends if we can find that our responses always contain certain objects.

Also, reponse should have a `completed` function which just returns a boolean for success of running the command.

Reply:
On second thought, I kinda like the interface you have now. I just worry that the entities are supposed to be higher level than being used within a command or response ya know?